.gitignore

```diff
@@ -6,6 +6,7 @@
 *.dylib
 *.golangci.yml
 *.swp
+*.pprof
```

Comment:
Not sure what is this changing ?

Reply:
This is a profiler data file, no need to commit those.